extensions/file/src/main/java/org/mule/extension/file/internal/FileOperations.java

```diff
@@ -162,14 +162,15 @@ public void write(@UseConfig FileConnectorConfig config, @Connection FileSystem
    * @param targetPath              the target directory where the file is going to be copied
    * @param createParentDirectories whether or not to attempt creating any parent directories which don't exists.
    * @param overwrite               whether or not overwrite the file if the target destination already exists.
+   * @param renameTo                the new file name, {@code null} if the file doesn't need to be renamed
    * @throws IllegalArgumentException if an illegal combination of arguments is supplied
    */
   @Summary("Copies a file in another directory")
```

Comment:
change this. The whole point of this change is that you can copy it in the same directory.